tests/js/spec/views/alerts/create.spec.jsx

```diff
@@ -269,30 +267,23 @@ describe('ProjectAlertsCreate', function () {
       await selectEvent.select(screen.getAllByText('all')[0], ['any']);
 
       // Change name of alert rule
-      fireEvent.change(screen.getByPlaceholderText('My Rule Name'), {
-        target: {value: 'My Rule Name'},
-      });
+      userEvent.paste(screen.getByPlaceholderText('My Rule Name'), 'My Rule Name');
```

Comment:
since `userEvent.type` does not have a very good performance  + this spec.jsx test was timing out 5000ms + the goal of these tests is not 100% on how the user interacts with the fields, but whether the form will be saved and updated...

I think it's ok to use `userEvent.paste ` here .

We can always revisit this test later and see if improvements can be made

Reply:
👍  I think this is great information to have as a code comment!